Language/Variables/Data Types/String/Functions/startsWith.adoc

```diff
@@ -26,16 +26,17 @@ Tests whether or not a String starts with the characters of another String.
 === Syntax
 `myString.startsWith(myString2)`
 
+
 [float]
 === Parameters
-`myString, myString2`: a variable of type String
+`myString, myString2`: a variable of type `String`.

```

Comment:
_`myString, myString2`: variables of type `String`._
or could be separated like you did above.